src/pytomography/utils/spatial.py

```diff
@@ -141,4 +141,78 @@ def unpad_object_z(object: torch.Tensor, pad_size: int):
         torch.Tensor[batch_size, Lx, Ly, Lz]:Unpadded object tensor.
     """
     
-    return object[:,:,pad_size:-pad_size]
\ No newline at end of file
+    return object[:,:,pad_size:-pad_size]
+
+def plot_slices(volume, title, batch_idx=0):
+    print(f"Volume shape: {volume.shape}")  # Debugging print
+    batch_volume = volume[batch_idx]
+    print(f"Batch volume shape: {batch_volume.shape}")  # Debugging print
+    
+    fig, axes = plt.subplots(1, 3, figsize=(15, 5))
+    slices = [
+        np.flip(batch_volume[batch_volume.shape[0] // 2, :, :].T, axis=1),  # Mirrored Sagittal
+        # batch_volume[batch_volume.shape[0] // 2, :, :].T,  # Mirrored Sagittal
+        batch_volume[:, batch_volume.shape[1] // 2, :].T,   # Coronal
+        batch_volume[:, :, batch_volume.shape[2] // 2].T    # Axial
+    ]
+    
+    for ax, slice_img, label in zip(axes, slices, ['HLA', 'SA', 'VLA']):
+        print(f"{label} slice shape: {slice_img.shape}")  # Debugging print
+        ax.imshow(slice_img, cmap='jet')
+        ax.set_title(label)
+        ax.axis('on')
+    
+    plt.suptitle(title)
+    plt.tight_layout()
+    plt.show()
+
+
+
+def object_reorient(volume, tx, ty, tz, beta, alpha, gamma):
```

Comment:
Add object types to input arguments. Rename to `euler_angle_transform`

Reply:
All input arguments are pytorch tensors

Reply:
Rename volume to object

Reply:
Done.